src/test/java/wolox/training/models/BookTest.java

```diff
@@ -0,0 +1,184 @@
+package wolox.training.models;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import wolox.training.repositories.BookRepository;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class BookTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @MockBean
```

Comment:
You shouldn't be mocking the repository in the DataJpaTests... given these test are actually for testing the repositories themselves.